### ISSUE TITLE
gatsby info not working

### ISSUE THREAD

**Issue by ukch** [gatsbyjs/gatsby#8502]
## Description
When I run `gatsby info` or `gatsby info --clipboard`, I see an error.

### Steps to reproduce

My gatsby config:
```javascript
require("dotenv").config({
    path: `.env.${process.env.NODE_ENV}`,
});
module.exports = {
    siteMetadata: {
        title: "MY SITE TITLE",
        api: {
            url: process.env.API_URL,
        },
    },
    plugins: [
        "gatsby-plugin-postcss",
        "gatsby-plugin-react-helmet",
    ]
}
```

### Expected result

Gatsby info should be displayed

### Actual result

```
$ gatsby info
error Cannot read property '1' of null


  TypeError: Cannot read property '1' of null
  
  - envinfo.js:1 a.run.then.e
    [lib]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:97255
  
  - util.js:16 tryCatcher
    [lib]/[gatsby-cli]/[bluebird]/js/release/util.js:16:23
  
  - promise.js:512 Promise._settlePromiseFromHandler
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:512:31
  
  - promise.js:569 Promise._settlePromise
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:569:18
  
  - promise.js:614 Promise._settlePromise0
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10
  
  - promise.js:694 Promise._settlePromises
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18
  
  - async.js:138 _drainQueueStep
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:138:12
  
  - async.js:131 _drainQueue
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:131:9
  
  - async.js:147 Async._drainQueues
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:147:5
  
  - async.js:17 Immediate.Async.drainQueues [as _onImmediate]
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:17:14
  

error UNHANDLED REJECTION


  TypeError: Cannot read property '1' of null
  
  - envinfo.js:1 a.run.then.e
    [lib]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:97255
  
  - util.js:16 tryCatcher
    [lib]/[gatsby-cli]/[bluebird]/js/release/util.js:16:23
  
  - promise.js:512 Promise._settlePromiseFromHandler
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:512:31
  
  - promise.js:569 Promise._settlePromise
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:569:18
  
  - promise.js:614 Promise._settlePromise0
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10
  
  - promise.js:694 Promise._settlePromises
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18
  
  - async.js:138 _drainQueueStep
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:138:12
  
  - async.js:131 _drainQueue
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:131:9
  
  - async.js:147 Async._drainQueues
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:147:5
  
  - async.js:17 Immediate.Async.drainQueues [as _onImmediate]
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:17:14
```

### Environment

For obvious reasons I can't paste the full output of `gatsby info`, but here are some details:
```
$ uname -sr
Linux 4.16.0-2-amd64
$ which node
/usr/local/nvm/versions/node/v9.2.1/bin/node
$ node --version
v9.2.1
$ which gatsby
/usr/local/nvm/versions/node/v9.2.1/bin/gatsby
$ gatsby --version
2.4.1
```

**Comment by kakadiadarpan** [comment]
@ukch is this your local machine or server?

**Comment by ukch** [comment]
@kakadiadarpan local machine. I use Debian Linux as my development environment.

**Comment by nakedible** [comment]
I have this bug as well

**Comment by stefanprobst** [comment]
This works fine for me in a Debian stretch VM. What version of Debian are you using?

**Comment by stefanprobst** [comment]
Tracked this down: This is a bug in the `envinfo` package which throws when there is no `VERSION` field in `/etc/os-release`.

**Comment by nakedible** [comment]
The bug has already been fixed in `envinfo`: https://github.com/tabrindle/envinfo/issues/51

**Comment by nakedible** [comment]
But apparently there is no release yet which includes the fix, as latest 5.10.0 does not.

**Comment by thorn0** [comment]
A new version of `envinfo` [has been released](https://github.com/tabrindle/envinfo/releases/tag/5.12.1). Currently, the latest one is 5.12.1. `gatsby-cli` has ^5.8.1 in its dependencies. So it should be enough to update the dependencies (e.g. run `npm update`) for the issue to go away.

@ukch @nakedible @stefanprobst Can you confirm that it's been fixed?
